api/schema/definition.json

```diff
@@ -0,0 +1,33 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Definition",
+  "type": "object",
+  "properties": {
+    "title": {
+      "type": "string"
+    },
+    "metadata": {
+      "type": "object",
+      "properties": {
+        "@type": { "type": "string" },
+        "language": { "type": "string" },
+        "modified": { "type": "string" },
+        "name": { "type": "string" },
+        "title": { "type": "string" }
+      }
+    },
+    "definitions": {
```

Comment:
This may be a problem if we want to use jsonschema because they already use this key word name so they can link references. It doesn't output any error or warning and the docs doesn't say you can't use it for anything other than what they use it for, but could lead to mix-ups down the road if the json turns out to be more complicated and references are used.